lib/realtime_web/channels/realtime_channel/presence_handler.ex

```diff
@@ -17,23 +18,28 @@ defmodule RealtimeWeb.RealtimeChannel.PresenceHandler do
   alias RealtimeWeb.Presence
   alias RealtimeWeb.RealtimeChannel.Logging
 
-  @spec call(map(), Phoenix.Socket.t()) ::
+  @spec call(map() | :sync_presence, Phoenix.Socket.t()) ::
```

Comment:
Perhaps have those as two different functions, `call(map())` and then `sync(socket)`?